sections/components.tex

```diff
@@ -0,0 +1,321 @@
+% !TeX spellcheck = en_US
+\addsection{Components}{\skills/luck.png}
+
+This is a visual guide of what is what, and shows components from every expansion together. Please refer to the official component lists for the exact numbers and pictures of all individual components.
```

Comment:
I like the idea, but I think the wording could be improved. Can't offer anything better yet.